### ISSUE TITLE
Inclusão de metadados adicionais (OID)

### ISSUE THREAD

**Issue by raiseppant** [demoiselle/signer#255]
Fala Pessoal!
Primeiramente quero parabenizar a iniciativa de implementar e disponibilizar este projeto para comunidade.

Estou precisando incluir no CAdes/PAdes, os metadados relacionados  aos dados médicos, conforme orientado no link abaixo:
https://assinaturadigital.iti.gov.br/duvidas/#1587761771301-8f0416f4-c42c .

Para eu incluir os OIDS no signer terei que implementar a interface IOIDExtensionLoader ? Como foi feita com a classe DefaultExtensionLoader? 

Ou existe uma outra forma?

Obrigado

**Comment by esaito** [comment]
Sim, pode ter como base ICPBrasilExtensionLoader e veja também o pacote org.demoiselle.signer.core.oid

**Comment by esaito** [comment]
Estamos para lançar a versão 2.6.0 ainda essa semana, se acha que pode fazer um pullrequest até o fim desta semana podemos aguardar.

**Comment by sampaiohelton** [comment]
Pessoal estou com a mesma demanda de incluir os metadados de dados médicos nos relatórios, prescrições conforme o link https://assinaturadigital.iti.gov.br/duvidas/#1587761771301-8f0416f4-c42c , alguém tem alguma pistas de como implementar isso nos documentos assinados? 
Obrigado!
